src/cli/cmds/fleet.toit

```diff
@@ -751,3 +819,18 @@ create-reference parsed/cli.Parsed config/Config cache/Cache ui/Ui:
   with-pod-fleet parsed config cache ui: | fleet/Fleet |
     fleet.write-reference --path=output
     ui.info "Created reference file $output."
+
+migration-start parsed/cli.Parsed config/Config cache/Cache ui/Ui:
+  broker-name := parsed["broker"]
+
+  with-devices-fleet parsed config cache ui: | fleet/FleetWithDevices |
+    new-broker := get-server-from-config config ui --name=broker-name
+    fleet.migration-start --broker-config=new-broker
+    ui.info "Started migration to broker $broker-name."
```

Comment:
Maybe add information on how to check the status right here?

Reply:
done.